target/scripts/helpers/log.sh

```diff
@@ -67,35 +66,42 @@ function _log() {
   case "${1}" in
     ( 'trace' )
       [[ ${LEVEL_AS_INT} -ge 5 ]] || return 0
-      MESSAGE+="  ${CYAN}TRACE  "
+      LOG_COLOR='CYAN'
+      LOG_LEVEL_NAME='TRACE'
       ;;
 
     ( 'debug' )
       [[ ${LEVEL_AS_INT} -ge 4 ]] || return 0
-      MESSAGE+="  ${PURPLE}DEBUG  "
+      LOG_COLOR='PURPLE'
+      LOG_LEVEL_NAME='DEBUG'
       ;;
 
     ( 'info' )
       [[ ${LEVEL_AS_INT} -ge 3 ]] || return 0
-      MESSAGE+="   ${BLUE}INF   "
+      LOG_COLOR='BLUE'
+      LOG_LEVEL_NAME='INFO '
       ;;
 
     ( 'warn' )
       [[ ${LEVEL_AS_INT} -ge 2 ]] || return 0
-      MESSAGE+=" ${LYELLOW}WARNING "
+      LOG_COLOR='LYELLOW'
+      LOG_LEVEL_NAME='WARN '
       ;;
 
     ( 'error' )
       [[ ${LEVEL_AS_INT} -ge 1 ]] || return 0
-      MESSAGE+="  ${LRED}ERROR  " ;;
+      LOG_COLOR='LRED'
+      LOG_LEVEL_NAME='ERROR'
+      ;;
 
     ( * )
       _log 'error' "Call to '_log' with invalid log level argument '${1}'"
       return 1
       ;;
   esac
 
-  MESSAGE+="${RESET}]  ${2}"
+  MESSAGE="$(date --iso-8601='seconds')  ${!LOG_COLOR}${LOG_LEVEL_NAME}${RESET}  "
+  MESSAGE+="$(basename "${0}" || echo 'unknown'): ${2}"
```

Comment:
Technically the date should be `date --rfc-3339='seconds'` (_`ns` would be more correct, but I'm not sure how you restrict the precision to 6 digits which is expected, ns precision is optional though_)

Reply:
> Do you have an example of what this looks like? (_yes I'm that lazy_ 😄 )

![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/8c3cac63-6b81-4cf6-a471-d2b1dd41fdd9)

Not so bad, if someone was trying to parse these logs and separate them, they'll need to go based on two columns to match as DMS logs now.

Still the added verbosity of the timestamp (_and a bit to the script name, but that's a little helpful_), I'm not sure about. Adjusting the `rsyslog` config won't adjust the timestamp format here since this isn't being managed via `rsyslog`. If we had some consistency one could better parse the logs and filter/format them as they see fit.